src/useObserver.ts

```diff
@@ -20,6 +19,8 @@ export function useObserver<T>(fn: () => T, baseComponentName: string = "observe
         return fn()
     }
 
+    const [objectRetainedByReact] = React.useState({})
```

Comment:
So this is essentially an identity of component instance, that's clever :)

Reply:
yes :)